gulpfile.js

```diff
@@ -0,0 +1,10 @@
+var gulp = require('gulp');
+var replace = require('gulp-replace');
+var packageJson = require('./package.json');
+var bowerJson = require('./bower.json');
+
+gulp.task('propagate-version', function() {
+  gulp.src(['./bower.json', './juicy-filedrop.html'])
+    .pipe(replace(bowerJson.version, packageJson.version))
```

Comment:
are you sure it will not replace accidentaly other versions? 
For example, we are updating to `0.0.2`, the previous bower.json looked like:
```js
{
"dependencies":{
   "mydep": ">=0.0.1"
},
"version": "0.0.1"
}
```
This replace will bump dependency version, won't it?